CHANGELOG.md

```diff
@@ -24,6 +24,9 @@ This project adheres to [Semantic Versioning](http://semver.org/).
   and nix::Error::UnsupportedOperation}`
   ([#614](https://github.com/nix-rust/nix/pull/614))
 - Added `cfmakeraw`, `cfsetspeed`, and `tcgetsid`. ([#527](https://github.com/nix-rust/nix/pull/527))
+- Added a `PtraceSyscall` variant to `nix::sys::wait::WaitStatus`
```

Comment:
I think this would be more clear if it stated something like "`wait`/`waitpid` now support `PTRACE_SYSCALL` through the new `WaitStatus::PtraceSyscall(pid)` type." Your current comment explains what changed in the code, but the CHANGELOG should be geared towards what changed in user-facing functionality, which my above suggestion does.